girder/api/rest.py

```diff
@@ -1057,10 +1057,22 @@ class boundHandler(object):  # noqa: class name
     ``self.boolParam``, ``self.requireParams``, etc.
     """
     def __init__(self, ctx=None):
-        self.ctx = ctx or _sharedContext
+        if callable(ctx):  # Used as a raw decorator
```

Comment:
The only concern here would be that someone might pass a `ctx` variable that was actually callable, i.e. they had some Resource class that overrode `__call__` for whatever reason, which would make this case ambiguous.